Asterix/optics/coronagraph.py

```diff
@@ -817,3 +860,140 @@ def create_wrapped_vortex_mask(dim,
         phase_mask = phase
 
     return angles, phase_mask
+
+
+def prop_fpm_regional_sampling(pup,
```

Comment:
No action needed in this PR, but next time moving a block of code should not be done while also modifying that code... it is absolutely impossible for me to spot all the changes and my review is kinda useless here since I will miss potential issues. For next time.